docs/source/guides/getting_started/first_sub.rst

```diff
@@ -0,0 +1,13 @@
+Initial Subscription
+====================
+
+Your first subscription should look similar to the below:
```

Comment:
Should mention something like "Using our config built in <ref>, your first subscription..."